pyTigerGraph/pyTigerGraphLoading.py

```diff
@@ -49,8 +49,50 @@ def runLoadingJobWithFile(self, filePath: str, fileTag: str, jobName: str, sep:
         if logger.level == logging.DEBUG:
             logger.debug("params: " + self._locals(locals()))
 
+        data = open(filePath, 'rb').read()
+        res = self.runLoadingJobWithData(data, fileTag, jobName, sep, eol, timeout, sizeLimit)
+
+        logger.info("exit: runLoadingJobWithFile")
+                
+        return res
+
+    def runLoadingJobWithData(self, data: str, fileTag: str, jobName: str, sep: str = None,
+            eol: str = None, timeout: int = 16000, sizeLimit: int = 128000000) -> Union[dict, None]:
+        """Execute a loading job with the given data string.
+
+        The data string will be posted to the TigerGraph server and the value of the appropriate
+        FILENAME definition will be updated to point to the data received.
+
+        NOTE: The argument `USING HEADER="true"` in the GSQL loading job may not be enough to
+        load the file correctly. Remove the header from the data file before using this function.
```

Comment:
the comment is confusing, same for runLoadingJobWithFile

So header should be removed before calling these two functions. If loading job still has Using header=true, will the first line be ignored?

Reply:
Actually I'd prefer to support HEADER=true in these 2 functions hence user can provide the parameters according to the loading job. @parkererickson-tg do you have any background information on why it might not loading correctly with HEADER specified?

Reply:
It is a long-time bug in the ddl system

Reply:
header=False will be required in df.to_csv() in this case.

Reply:
ok, make it explicit that USING HEADER=false in loading job definition.

Reply:
HEADER=false is actually the default behavior

Reply:
Right. I mean user should not set USING HEADER=true in loading job in this case? otherwise they will lose 1 row?